### PR TITLE
Adding oc_group to lib_openshift.

### PR DESCRIPTION
**Pull request by kwoodson**

### CODE REVIEW

**Comment by ashcrow** [comment]
Looks like there are some ``flake8`` and ``pylint`` stuff that needs updating. Other than that it looks good.

File: roles/lib_openshift/src/ansible/oc_group.py [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+            debug=dict(default=False, type='bool'),
+            name=dict(default=None, type='str'),
+            namespace=dict(default='default', type='str'),
+            # addind users to a group is handled through the oc_users module
+            #users=dict(default=None, type='list'),
+        ),
+        supports_check_mode=True,
+    )
+
+    gconfig = GroupConfig(module.params['name'],
```
Comment (ashcrow) [inline roles/lib_openshift/src/ansible/oc_group.py]: Looks like this snuck in again. :smile:

File: roles/lib_openshift/src/class/oc_group.py [hunk 2]
```diff
@@ -0,0 +1,152 @@
...
+            return True
+
+        return False
+
+    def get(self):
+        '''return group information '''
+        result = self._get(self.kind, self.config.name)
+        if result['returncode'] == 0:
+            self.group = Group(content=result['results'][0])
+        elif 'groups \"%s\" not found' % self.config.name in result['stderr']:
```
Comment (ashcrow) [inline roles/lib_openshift/src/class/oc_group.py]: nit(not_required): I'd recommend moving away from ``%s``'s and towards string formatting like so:

(assuming the \'s are required in the output check)

```python
elif 'groups \"{}\" not found'.format(self.config.name) in result['stderr']:
```

**Comment by ashcrow** [comment]
@kwoodson don't worry about coveralls :smile:

**Comment by kwoodson** [comment]
@ashcrow, heh, thanks.

**Pull request merged**